File: boj10430.cpp
//boj10430 나머지
#include <stdio.h>
int main(void) {
	int A, B, C;
	scanf("%d %d %d", &A, &B, &C);
    printf("%d\n", (A+B)%C );
	printf("%d\n", (A%C+B%C)%C );
    printf("%d\n", (A*B)%C );
    printf("%d", (A%C*B%C)%C );
	return 0;
}

File: boj10430.java
//boj10430 나머지
import java.util.*;

class Main{
    public static void main(String arg[]){
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();
        System.out.printf("%d\n%d\n%d\n%d",(a+b)%c,(a%c+b%c)%c,(a*b)%c,(a%c*b%c)%c);
    }
}
